rwengine/src/script/modules/GameModule.inc

```diff
@@ -984,6 +984,9 @@ void game_clear_area(const ScriptArguments& args)
 
 	for(auto& p : gw->pedestrianPool.objects)
 	{
+		if (p.second->getLifetime() == GameObject::PlayerLifetime) {
```

Comment:
Add a comment that this is a hack and we should check something like the traffic flag or a script reference count instead?